keyboards/sawnsprojects/re65/keymaps/vial/vial.json

```
@@ -0,0 +1,212 @@
+{
+  "name": "RE65",
+  "vendorId": "0x534C",
+  "productId": "0x0C65",
+  "matrix": {
+    "rows": 5,
+    "cols": 15
+  },
+  "keycodes": ["qmk_lighting"],
+  "menus": ["qmk_rgblight"],
```

```suggestion
  "lighting": "qmk_rgblight",
```
These v3 changes aren't taken into account for Vial.